### ISSUE TITLE
Tutorial doesn't work on Mac ARM 

### ISSUE THREAD

**Issue by dprotaso** [concourse/concourse#8988]
I'm testing out the tutorial and I'm noticing the hello world pipeline is giving me an error

> run check: find or create container on worker ff119f8784a5: starting task: new task: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: can't copy bootstrap data to pipe: write init-p: broken pipe: unknown

I'm on an Apple M1 Pro testing the latest concourse (v7.11.2)  - Docker Desktop 4.32.0 (157355)

**Comment by PhilippMatthes** [comment]
I deployed concourse via helm and switching to the `houdini` worker runtime fixed this issue for me. See https://github.com/concourse/concourse-chart/blob/1199fdee355a49457076fd742e31729c8c605329/values.yaml#L1708

**Comment by ragaskar** [comment]
For others arriving here via search: if you're following https://concourse-ci.org/quick-start.html (which describes deployment via docker-compose vs helm), then there's a helpful note in the docker-compose.yml that you may have missed: 

```
 # For ARM-based machine, change the Concourse runtime to "houdini"
 CONCOURSE_WORKER_RUNTIME: "containerd"
```

I can validate that changing the CONCOURSE_WORKER_RUNTIME to "houdini" as suggested and then re-starting the processes appears to get one past the problem described in this issue (at least on an M1 machine running Sonoma).

**Comment by ragaskar** [comment]
Also, if you're affected by this issue, you might want to have a look at https://github.com/concourse/git-resource/issues/404 (git-resource encounters permission issues on houdini that can be worked around w/ some explicit repo config -- sounds like houdini containerization is "weird" and so maybe that's why it happens).

**Comment by ragaskar** [comment]
lol, ok, that actually didn't get me much further, I started choking on ability to mount. 

I wound up switching tactics, using an arm64 image that someone else has built got me much further: https://github.com/concourse/concourse/issues/8719#issuecomment-1676821087

I am still having issues running the concourse/oci-build-task ... which I suspect might also be arm related ...

**Comment by ragaskar** [comment]
For others: 

I spent a fair amount of time trying to get the concourse/oci-build-task to work. I _still think_ it's likely ARM-related (see a similar problem people had with docker-in-docker builds with earthly: https://github.com/earthly/earthly/issues/3937). The linked issue suggests that maybe one can pass an explicit platform argument to buildkit, but this argument doesn't seem plumbed in oci-build-task (if it's helpful, one can intercept into the failed build task and run `DEBUG=1 build` to get a look at how buildkit is called. 

I suspect building an arm64 version of oci-build-task and using it instead would likely work, but I didn't get that far. I only needed oci-build-task to match an image build I was doing in my "prod" pipeline that I am attempting to test locally, and that I am doing that build as a task vs. simply retrieving an image is already a workaround. 

I solved my problem of "need an image for my builds" by spinning up a local docker registry (see [here](https://hub.docker.com/_/registry) and also, note this fun fact: airplay seems to slurp up port 5000 on osx), building the image I wanted to build with oci-build-task and pulling it directly as part of the docker task. This seems to work fine. I bet one could also choose to take this approach with the oci-build-task itself (someone has built an arm64 image of oci-build-task but it hasn't been updated in many years).
